libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -104,6 +104,21 @@ class YulUtilFunctions
 	std::string overflowCheckedUIntSubFunction();
 
 	std::string arrayLengthFunction(ArrayType const& _type);
+
+	/// @returns the name of a function that resizes an array
+	/// signature: (array, newLen)
+	std::string resizeDynamicArrayFunction(ArrayType const& _type);
+
+	/// @returns the name of a function that will clear the storage area given
+	/// by the start and end parameters
+	/// signature: (start, end)
+	std::string clearStorageRangeFunction(Type const& _type);
+
+	/// Returns the name of a function that will convert a given length to the
```

Comment:
Please be more specific about what it does for storage. Does it only return the number of slots? Does it round up?